lib/matplotlib/backends/backend_qt5.py

```diff
@@ -254,8 +254,11 @@ def mousePressEvent(self, event):
         # flipy so y=0 is bottom of canvas
         y = self.figure.bbox.height - event.pos().y()
         button = self.buttond.get(event.button())
+        modifiers = "+".join([name for name, mod_key, qt_key in MODIFIER_KEYS
+                if (int(event.modifiers()) & mod_key) == mod_key])
```

Comment:
Maybe make modifiers a `set` instead of a string like `"ctrl+alt"`? Also, I think this could be simplified to
```
name for name, mod_key, _ in MODIFIER_KEYS if event.modifiers() & mod_key
```